charts/values.yaml.example

```diff
@@ -0,0 +1,292 @@
+  ############################
+  #      ### Example of profile ###
+  #      profiles: |
+  #        nameofprofile:
+  #          frequency: ###number_of_seconds###
+  #          varBinds:
+  #            - ['SNMPv2-MIB', 'sysDescr']
+  #            - ['SNMPv2-MIB', 'sysName', 0]
+  #            - 1.2.3.2.1.*
+  #            - 1.3.6.1.2.1.2.1.0
+  ############################
+  #      ### Example of additional metric fields ###
+  #      additionalMetricField: |
+  #        - profile
+  traps:
+    config: |
+      communities:
+        v2:
+          - public
+#            - "my-area"
+#          v1:
+#            - public
+#            - "my-area"
+#          v3:
+#            - userName: snmpv3test
+#              authKey: AuthPass1
+#              privKey: PrivPass2
+#              securityEngineId: 8000000004030201
+#            - userName: snmpv3test2
+#              authProtocol: SHA
+#              authKey: AuthPass11
+#              privProtocol: aes
+#              privKey: PrivPass22
+#              securityEngineId: 8000000004030202
+#            - userName: snmpv3test3
+#              securityEngineId: 8000000004030203
+#            - userName: testfdse
+#              authKey: testfdse
+#              privKey: testfdse
+#              securityEngineId: 8000000903bc16f5802780
+#############################################################################
+### Please do not modify below values, unless you know what you're doing! ###
+#############################################################################
+mongodb:
+  ## @section Global parameters
+  ## Global Docker image parameters
+  ## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+  ## Current available global Docker image parameters: imageRegistry, imagePullSecrets and storageClass
+
+  ## @section Common parameters
+  ##
+
+  ## @param fullnameOverride String to fully override mongodb.fullname template
+  ##
+  fullnameOverride: "sc4snmp-mongodb"
+  ## @param clusterDomain Default Kubernetes cluster domain
+  ##
+  clusterDomain: cluster.local
+
+  ##
+  auth:
+    ## @param auth.enabled Enable authentication
+    ## ref: https://docs.mongodb.com/manual/tutorial/enable-authentication/
+    ##
+    enabled: false
+
+  ## MongoDB&reg; containers' resource requests and limits.
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param resources.limits The resources limits for MongoDB&reg; containers
+  ## @param resources.requests The requested resources for MongoDB&reg; containers
+  ##
+  resources:
```

Comment:
What about commenting out this resources? Like:
```
resources: {}
  #  limits:
  #    cpu: 500m
  #   memory: 768Mi
  #  requests:
  #    cpu: 100m
  #    memory: 512Mi
```

Reply:
I was testing it on EC2 medium machine (with `values.yaml`) and had hard time trying to adjust cpu to have all of the pods up. Commenting out the fixed resources for mongodb fixed the issue.